src/__tests__/enhancer-test.js

```diff
@@ -7,41 +8,54 @@ import React, {Component} from 'react';
 
 describe('Enhancer', () => {
   it('sets up initial state', () => {
-    class Composed extends Component {}
+    class Composed extends Component {
+      render() {
+        return <div />;
+      }
+    }
     const Enhanced = Enhancer(Composed);
 
-    const instance = new Enhanced();
```

Comment:
can't do tests like this as class components are now wrapped in a HoC to inject context